docs/whatsnew/0.8.rst

```diff
@@ -0,0 +1,239 @@
+
+TimeSeries objects can have more than one column::
+
+    >>> goes.columns
+    ['xrsa', 'xrsb']
+
+and have convenient plotting methods::
+
+    >>> goes.peek()
+
+TimeSeries objects have a 'meta' property that stores the metadata of the timeseries::
+
+    >>> goes.meta
+    |-------------------------------------------------------------------------------------------------|
+    |TimeRange                  | Columns         | Meta                                              |
+    |-------------------------------------------------------------------------------------------------|
+    |2011-06-06 23:59:59.961999 | xrsa            | simple: True                                      |
+    |            to             | xrsb            | bitpix: 8                                         |
+    |2011-06-07 23:59:57.631999 |                 | naxis: 0                                          |
+    |                           |                 | extend: True                                      |
+    |                           |                 | date: 26/06/2012                                  |
+    |                           |                 | numext: 3                                         |
+    |                           |                 | telescop: GOES 15                                 |
+    |                           |                 | instrume: X-ray Detector                          |
+    |                           |                 | object: Sun                                       |
+    |                           |                 | origin: SDAC/GSFC                                 |
+    |                           |                 | ...                                               |
+    |-------------------------------------------------------------------------------------------------|
+
+and the data can be accessed::
+
+    >>> goes.data
+                                        xrsa          xrsb
+    2011-06-06 23:59:59.961999  1.000000e-09  1.887100e-07
+    2011-06-07 00:00:02.008999  1.000000e-09  1.834600e-07
+    2011-06-07 00:00:04.058999  1.000000e-09  1.860900e-07
+    ...
+    2011-06-07 23:59:55.584999  1.000000e-09  1.624800e-07
+    2011-06-07 23:59:57.631999  1.000000e-09  1.598500e-07
+
+Data sources that do not provide FITS files need to have a 'source' keyword to help
+with the identification and interpretation of the data::
+
+    >>> eve = sunpy.timeseries.TimeSeries(sunpy.data.sample.EVE_TIMESERIES, source='EVE')
+
+.. _whatsnew-0.8-differential_rotation:
+
+Differential rotation of maps
+=============================
+
+Maps can now be transformed using solar differential rates::
+
+    >>> import sunpy.map
+    >>> import sunpy.data.sample
+    >>> from sunpy.physics.differential_rotation import diffrot_map
+    >>> aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+    >>> dr_map = diffrot_map(aia_map, -4 * u.day)
```

Comment:
Seeing it being used... shouldn't `diffrot_map` be a method of `map`? Then it could be used as:
```python
   >>> aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
   >>> dr_map = aia_map.diffrot_map(-4 * u.day) # Dr Map! 
```
or we could offer both as numpy does with `np.max(x)` or `x.max()`

Also maybe it should accept a time instead of a quantity...